src/routes/tabs/VideoPlayerTab.svelte

```diff
@@ -0,0 +1,187 @@
+<script lang="ts">
+    import img from "$lib/images/strommingPog.png";
+	import vid from "$lib/videos/video.mp4";
+
+	import { Grid, gridHelp } from "$lib/js/svelte-grid";
+
+    let items = [
+    {
+		6: gridHelp.item({
+			x: 0,
+			y: 0,
+			w: 4,
+			h: 5,
+		}),
+		//id: id(),
+		id: "Box1",
+    },
+    {
+		6: gridHelp.item({
+			x: 4,
+			y: 0,
+			w: 2,
+			h: 2,
+		}),
+		id: "Box2",
+    },
+	{
+		6: gridHelp.item({
+			x: 4,
+			y: 2,
+			w: 2,
+			h: 2,
+		}),
+		// This makes a random ID
+		//id: id(),
+		id: "Box3"
+	},
+  ];
+
+  let boll = false;
+  const cols = [[1200, 6]];
+
+	// TODO: export to type-file
+	type boxItem = {
+		6: any,
+		id: string,
+	}
+
+	function toggleBoll() {
+		boll = !boll;
+	}
+
+	function remove(item: boxItem) {
+  		items = items.filter((value) => value.id !== item.id);
+	}
+
+	function add() {
+		let newItem = {
+			6: gridHelp.item({
+			w: 2,
+			h: 2,
+			x: 0,
+			y: 0,
+			}),
+			id: id(),
```

Comment:
I think this might be my error from earlier? Or is it new?

Reply:
i copied your code to the tab :)

Reply:
i moved the id that was in page.svelte here now. should be fine